kolibri/content/views.py

```diff
@@ -15,6 +18,17 @@
 
 class ZipContentView(View):
 
+    @xframe_options_exempt
+    def options(self, request, *args, **kwargs):
+        """
+        Handles OPTIONS requests which may be sent as "preflight CORS" requests to check permissions.
+        """
+        response = HttpResponse()
+        response["Access-Control-Allow-Origin"] = "*"
+        response["Access-Control-Allow-Methods"] = "GET, OPTIONS"
```

Comment:
We might also need `Access-Control-Allow-Headers`.

This is what I get for the file from earlier today, running the code from this PR:
http://35.196.143.130/learn/#/topics/c/10f41b318dc546dabe0a596aaa537a53

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://35.196.143.130/zipcontent/213988fd537e4a608e2b86be9632d92d.zip/xml/course_config.xml. (Reason: missing token ‘x-requested-with’ in CORS header ‘Access-Control-Allow-Headers’ from CORS preflight channel).

In FF and Chrome.